src/Cargo.toml

```diff
@@ -16,6 +16,14 @@ members = [
   "tools/rust-installer",
   "tools/cargo",
   "tools/rls",
+  "tools/rls/test_data/borrow_error",
+  "tools/rls/test_data/completion",
+  "tools/rls/test_data/find_all_refs",
+  "tools/rls/test_data/find_all_refs_no_cfg_test",
+  "tools/rls/test_data/goto_def",
+  "tools/rls/test_data/highlight",
+  "tools/rls/test_data/hover",
+  "tools/rls/test_data/rename",
```

Comment:
error:
```
thread '<unnamed>' panicked at 'could not create cargo workspace: current package believes it's in a workspace when it's not:
current:   /home/ncameron/rust2/src/tools/rls/test_data/highlight/Cargo.toml
workspace: /home/ncameron/rust2/src/Cargo.toml

this may be fixable by adding `tools/rls/test_data/highlight` to the `workspace.members` array of the manifest located at: /home/ncameron/rust2/src/Cargo.toml', src/libcore/result.rs:859
```